plugin/keyring_vault/keyring_vault-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+ADD_EXECUTABLE(keyring_vault-test
```

Comment:
I have checked hp_test1 and added this test app to make test. However, it leaves some keys undeleted in Vault when run - this is because it runs couple of threads in parallel and some of the keys get added/generate multiple times as remove threads can delete some of the keys in a mean time. Thus, vault used for this test may need to be cleared from time to time - quite rarely as it leaves around 10 keys undeleted per execution. The cleaning will be needed any way for mtr tests.